n2v/grid/grider.py

```diff
@@ -643,6 +643,50 @@ def on_grid_all(self, grid=None, cubic_grid=False, show_progress=False):
 
         return
 
+    def DFT_grid_to_fock(self, value, Vpot):
```

Comment:
Is it possible to do "dft_grid_to_fock" or "grid_to_fock"? just to keep the convention of functions start with lowercase and classes with uppercase.